db/data/20200612054906_copy_teleconsultation_phone_numbers_to_json.rb

```diff
@@ -1,13 +1,18 @@
 class CopyTeleconsultationPhoneNumbersToJson < ActiveRecord::Migration[5.2]
   def up
-    Facility.where(enable_teleconsultation: true).each do |facility|
-      facility.teleconsultation_phone_numbers = [{"isd_code" => facility.teleconsultation_isd_code,
-                                                  "phone_number" => facility.teleconsultation_phone_number}]
+    Facility.all.each do |facility|
```

Comment:
What's the bug? We had facilities where teleconsult was disabled, but they had phone numbers?

Reply:
That's also a possibility, but the bug here was that `teleconsultation_phone_numbers=` used to simply accept some JSON and set it, but now its a defined setter that failed to pick up string keys we're passing in here.